trie/trieStorageManager_test.go

```diff
@@ -36,6 +37,12 @@ func getNewTrieStorageManagerArgs() trie.NewTrieStorageManagerArgs {
 	}
 }
 
+// ErrChanWithLen extends the BufferedErrChan interface with a Len method
+type ErrChanWithLen interface {
```

Comment:
If we need this only in tests, we might change this interface to unexported.

Reply:
Done.